CefSharp.BrowserSubprocess.Core/RegisterBoundObjectHandler.h

```diff
@@ -96,8 +96,6 @@ namespace CefSharp
 
                             auto objectName = arguments[0]->GetStringValue();
 
-                            auto global = context->GetGlobal();
```

Comment:
Is `global` used in the outer scope? If not then remove the outer scope declaration instead.

Reply:
Good point. Removed.